### ISSUE TITLE
Be consistent with alt text

### ISSUE THREAD

**Issue by farishkash** [wakedork/f1-3-c2p1-colmar-academy#3]
Some of your images have alt text others don't.  Should be used for all.  There are more people using the web that are visually impaired that you realize.

Here is a video on a presentation from a blind programmer.

https://www.youtube.com/watch?v=iWXebEeGwn0
